docs/canonicalk8s/charm/howto/install/install-lxd.md

```diff
@@ -1,129 +1,35 @@
 # How to install to localhost/LXD
 
 The main [install instructions][install] cover most situations for installing
-{{product}} using a charm. However, there are two scenarios which
-require some extra steps. These are:
+{{product}} using a charm. However, using LXD requires special consideration.
 
-- deploying to the 'localhost' cloud
+Juju can leverage LXD by:
+
+- deploying to the default 'localhost' cloud
+- explicitly adding a Juju cloud that uses the ``lxd`` type
 - deploying to a container on a machine (i.e. when installing a bundle or using
   the 'to:' directive to install to an existing machine)
 
-The container running the charm, or more accurately, the LXD instance
-controlling the container, needs to have a particular configuration in order
-for the Kubernetes components to operate properly.
-
-
-## Apply the {{product}} LXD profile
-
-On the machine running the 'localhost' cloud, we can determine the existing
-profiles by running the command:
-
-```
-lxc profile list
-```
-
-For example, suppose we have created a model called `myk8s`. This will
-output a table like this:
+LXD privileged containers are no longer supported and some Kubernetes services,
```

Comment:
````suggestion
```{warning}
LXD privileged containers are no longer supported and some Kubernetes services, such as the Cilium CNI, cannot run inside unprivileged containers.
```
````